src/App/Components/Articles/getArticles/Article.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import propTypes from 'prop-types';
+import Rater from 'react-rater';
+import ReactHtmlParser from 'react-html-parser';
+import ReactImageFallback from 'react-image-fallback';
+import { Link } from 'react-router-dom';
+import countRating from '../../../../Helpers/countRating';
+
+export class Article extends Component {
+  render() {
+    const { data } = this.props;
+    const {
+      body, title, coverImage, createdAt, slug
+    } = data;
+    const { name } = data.Category;
+    const { profileImage, firstName, username } = data.User;
+    const { comments, reads, shares } = data.stats;
+    const { allUsers, statistics } = data.rating;
+    const ratings = countRating(statistics);
+
+    const date = moment(createdAt).format('MMM Do YY');
+    const img = 'https://ielektro.es/wp-content/uploads/2017/04/ventajas-comprar-LED.jpg';
+    const defaultProImage = 'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png';
+    const articleBody = body.length > 200 ? `${body.substring(0, 210)} ...` : body;
+    const articleTitle = title.length > 52 ? `${title.substring(0, 52)}...` : title;
+    const categoryColor = (name === 'LOVE' && '#ff1744') || (name === 'TECHNOLOGY' && '#3f51b5') || (name === 'MUSIC' && '#880e4f') || (name === 'BUSINESS' && '#2196f3') || (name === 'ART' && '#9c27b0') || '#afad99';
```

Comment:
@Ramadhan0 can this be placed in a constants file which can be imported and re-used?

Reply:
here I am updating the class names of my div and i am the only one who needs this @MemunaHaruna